shell/common/switches.cc

```diff
@@ -78,13 +80,13 @@ void PrintUsage(const std::string& executable_name) {
   std::cerr << std::string(column_width, '-') << std::endl;
 }
 
-const char* FlagForSwitch(Switch swtch) {
+const ftl::StringView FlagForSwitch(Switch swtch) {
```

Comment:
Looking at this PR it seems I could keep the return type here and simply return `gSwitchDescs[i].flag.data()` on line 86, and pretty much the rest of the diff would go away. WDYT?